centreon/www/front_src/src/Header/Poller/translatedLabels.ts

```diff
@@ -24,7 +24,7 @@ export const labelDatabaseNotActive =
   'Some database poller updates are not active; check your configuration';
 export const labelDatabaseUpdatesNotActive = 'Database issue';
 export const labelPollerNotRunning = 'Pollers not running';
-export const labelPollerRunning = 'Everything Ok';
+export const labelPollerRunning = 'Everything OK';
 export const labelConfigurePollers = 'Configure pollers';
-export const labelAllPollers = 'All pollers:';
+export const labelAllPollers = 'All pollers :';
```

Comment:
```suggestion
export const labelAllPollers = 'All pollers:';
```

Reply:
can you explain ? should we have a space in French and not in English ? Because it was a CR from @cg-tw to add a space in the French version.

Reply:
@gael-boyenval No space in EN, a space in FR